src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageprisonvisitsorchestration/controller/OrchestrationSessionsController.kt

```diff
@@ -130,8 +130,20 @@ class OrchestrationSessionsController(private val visitSchedulerSessionsService:
       example = "dfs-wjs-eqr",
     )
     excludedApplicationReference: String? = null,
+    @Parameter(
+      description = "Advances the available visits slots sought from date by n days. Defaults to 0 if not passed.",
+    )
+    advanceFromDateByDays: Int?,
   ): List<AvailableVisitSessionDto> =
-    visitSchedulerSessionsService.getAvailableVisitSessions(prisonCode, prisonerId, sessionRestriction, visitors, withAppointmentsCheck ?: true, excludedApplicationReference)
+    visitSchedulerSessionsService.getAvailableVisitSessions(
+      prisonCode = prisonCode,
+      prisonerId = prisonerId,
+      requestedSessionRestriction = sessionRestriction,
+      visitors = visitors,
+      withAppointmentsCheck = withAppointmentsCheck ?: true,
+      excludedApplicationReference = excludedApplicationReference,
+      advanceFromDateByDays = advanceFromDateByDays ?: 0,
```

Comment:
no need if you have above

Reply:
I had to keep this but remove it from above.